pkg/kubectl/run.go

```diff
@@ -195,6 +247,11 @@ func (BasicPod) Generate(params map[string]string) (runtime.Object, error) {
 		return nil, err
 	}
 
+	command, err := GetBool(params, "command", false)
```

Comment:
Move this inside the `if len(args) > 0` clause as you already do in `BasicReplicationController`.